### ISSUE TITLE
Add coordinates to temperature plot

### ISSUE THREAD

**Issue by esseks** [BitsDevelopmentTeam/bits-server#7]
Temperature data is plotted without axis and the resulting plot is absolutely pointless:

![graph](https://f.cloud.github.com/assets/464720/1254311/d6cd4628-2b73-11e3-8431-2168e6660845.png)

This a serious interaction failure, at least in three points:
1. at a first glance, it is impossible to tell what the graph is actually showing.
2. it is hard to tell if the variations outlined are actually relevant (delta could be less than 1°C, but who knows?)
3. there is no data about the historical progression. For instance, look at the steep variation in the graph reproduced above: is the delta so relevant? Maybe we log temperature once every six months and we have one sample in winter and one in summer.

To summarize: that plot eats space and bandwidth without providing any useful piece of information to the final user.
